content/updates/serenity-updates-for-the-pim.json

```diff
@@ -0,0 +1,158 @@
+            "img": "/bundles/akeneocommunicationchannel/images/cards/associations-number.png",
+            "altImg": "Associations number inside the product form",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-04.html#product-associations-update"
+        },
+        {
+            "startDate": "May, 7th 2020",
+            "notification_duration": 7, 
+            "tags": ["updates"],
+            "title": "Track your destination connections",
+            "description": "You can now easily track the number of products that are sent to your destinations in the Connection dashboard.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/number-of-products-sent.png",
+            "altImg": "Destination connection chart",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-04.html#track-your-destination-connections"
+        },
+        {
+            "startDate": "May, 7th 2020",
+            "notification_duration": 7, 
+            "tags": ["updates"],
+            "title": "Connection dashboard updates",
+            "description": "The Connection dashboard has been improved with a new layout as well as the possibility to choose which connection you want to track.",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-04.html#connection-dashboard-updates"
+        },
+        {
+            "startDate": "May, 7th 2020",
+            "notification_duration": 7, 
+            "tags": ["updates"],
+            "title": "Rules engine new actions and improvements",
+            "description": "To extend the rules engine capabilities, we added two new actions, the possibility to name your rules and we also improved the rules format for associations.",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-04.html#rules-engine-new-actions-and-improvements"
+        },
+        {
+            "startDate": "May, 7th 2020",
+            "notification_duration": 7, 
+            "tags": ["updates"],
+            "title": "Data quality improvements",
+            "description": "To make sure all your products have enriched and consistent content, Data Quality Insights recommendations and grades are now also available for product models. Our spell-check feature also now supports additional languages.",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-04.html#data-quality-improvements"
+        },
+        {
+            "startDate": "April, 2nd 2020",
+            "notification_duration": 7, 
+            "tags": ["updates"],
+            "title": "Data quality improvements",
+            "description": "To help you craft the best product information ever, Data quality insights recommendations and grades are now available for variant products and our spell-check feature supports additional languages!",
```

Comment:
@LaureBro We have twice the same title. Sounds weird. Should we rename one of them?

Reply:
We could have: "Data Quality Insights now available for product models" and then "Data Quality Insights now available for product variants" and we don't mention the spell check in the title, what do you think?
OR  we rename one of them, like you suggested for instance with "Data quality enhancements"

Reply:
It feels weird not mentioning the spell-check.
What about one of the title saying "Data Quality Insights made even more powerful"?